### PR TITLE
standardize diff help message

### PR DESCRIPTION
**Pull request by poornas**
Fixes issue #2212 to standardize diff help message to be similar to other mc commands.

### CODE REVIEW

File: cmd/diff-main.go [hunk 1]
```diff
@@ -42,16 +42,12 @@ var diffCmd = cli.Command{
...
   {{.HelpName}} - {{.Usage}}
-
 USAGE:
   {{.HelpName}} [FLAGS] FIRST SECOND
 
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
-DESCRIPTION:
-  {{.Description}}
```
Comment (vadmeste) [inline cmd/diff-main.go]: @poornas, by removing description, we will lose this text:
`  Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. Objects of same name and size, but differ in contents are not noticed.` which is kind of important to know.

Can you you manage to add this information in USAGE ?

Comment (poornas) [inline cmd/diff-main.go]: done.

File: cmd/diff-main.go [hunk 2]
```diff
@@ -41,17 +41,13 @@ var diffCmd = cli.Command{
 	Before:      setGlobalsFromContext,
 	Flags:       append(diffFlags, globalFlags...),
 	CustomHelpTemplate: `NAME:
-  {{.HelpName}} - {{.Usage}}
-
+  {{.HelpName}} - {{.Usage}}{{.Description}}
```
Comment (vadmeste) [inline cmd/diff-main.go]: I think it is better to remove Description and modify Usage

File: cmd/diff-main.go [hunk 3]
```diff
@@ -34,24 +34,19 @@ var (
...
 // Compute differences between two files or folders.
 var diffCmd = cli.Command{
-	Name:        "diff",
-	Usage:       "Show differences between two folders or buckets.",
-	Description: "Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. Objects of same name and size, but differ in contents are not noticed.",
-	Action:      mainDiff,
-	Before:      setGlobalsFromContext,
-	Flags:       append(diffFlags, globalFlags...),
+	Name:   "diff",
+	Usage:  "Show differences between two folders or buckets. Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. Objects of same name and size, but differ in contents are not noticed.",
```
Comment (balamurugana) [inline cmd/diff-main.go]: I think below `Usage` is better
```
List objects with size difference or missing between two folders or buckets
```

In `Description` you could have what algo it does like
```
Diff only lists missing objects or objects with size differences between given folders or buckets. It *DOES NOT* check for content difference of objects those having same name and size.
```

I can even think of removing `OUTPUT` section and put the info into `Description` like
```
Diff only lists missing objects or objects with size differences between given folders or buckets. It *DOES NOT* check for content difference of objects those having same name and size.

Differences are marked in output by:
    > - indicates object missing in destination.
    < - indicates object missing in the source.
    ! - indicates object size differed in source and destination.
```

Comment (poornas) [inline cmd/diff-main.go]: @balamurugana, Description section is being removed here to make it consistent with other mc commands.

File: cmd/diff-main.go [hunk 4]
```diff
@@ -34,25 +34,21 @@ var (
...
   {{.HelpName}} [FLAGS] FIRST SECOND
 
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 DESCRIPTION:
-  {{.Description}}
-
-OUTPUT:
+  Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. objects of same name and size, but differing in content are not noticed.
```
Comment (harshavardhana) [inline cmd/diff-main.go]: We can take this verbatim description from the `diffCmd` initializer and keep the help template simpler and clean.

Comment (poornas) [inline cmd/diff-main.go]: done

**Comment by poornas** [comment]
ping..needs review

**Comment by deekoder** [comment]
@vadmeste  do you approve?

**Pull request merged**